### ISSUE TITLE
We need a full completed module for modern pages (Symfony) PS_1.7.5.1

### ISSUE THREAD

**Issue by Bossman1337** [PrestaShop/PrestaShop#12693]
**Is your feature request related to a problem? Please describe.**
The current documentation isn't clear (https://devdocs.prestashop.com/1.7/development/architecture/migration-guide/forms/)

https://github.com/PrestaShop/PrestaShop/issues/11926 - This isn't the reply i would like either. A) That is for a legacy module B) The Main Forums seem slow/inactive C) Stackoverflow, no one replies as they don't know either.

**Describe the solution you'd like**
A full "Modern" module that we can download/install and view the source. A module that adds a new field to the product admin page. When the admin product page is loaded, it will load this new fields data, also when the product is saved it will save that field data. Would also be good to render the result to a twig template.

**Additional context**
The Prestashop documentation isn't the best (It's okay, but seems rushed tbh)

I'm developing/testing out creating new modules for PS. Symfony isn't the problem. Mainly its the best practices we need to learn and how Symfony and PS mix together, ready for current and future versions of PS. Currently from the dev docs it looks like we need to work in "src/PrestaShopBundle..." but how does that relate to a "Module", hows the install handled, or are all files meant to be within the module folder?

Basically, if a PS dev could knock up a quick module for us to download (Described above) it will answer 100's of questions in one go 👍

**Comment by mickaelandrieu** [comment]
You may find some resources here: https://github.com/friends-of-prestashop

But imho, it shouldn't be us (ie the PrestaShop company) but you the developers that should improve the documentation according to your needs 👍
